app/assets/stylesheets/zeal-initiative-tracker.webflow.css

```diff
@@ -25,6 +25,7 @@ h3 {
   margin-bottom: 0px;
   font-size: 24px;
   font-weight: 700;
+ 
```

Comment:
Please remove empty lines within style block

Reply:
This issue was resolved